modules/README.md

```diff
@@ -4,20 +4,45 @@ Modules should be entirely self-contained into a subdirectory with the following
 
     ModuleName/
 
-        ajax/         - This directory contains any AJAX helper script used by the module. They should
-                        be invoked by using the AjaxHelper.php script in Loris
-
         css/          - Any necessary new CSS classes/files should go here
 
-        help/         - Contains a markdown file of help content for each page in this module
+        help/         - Contains a markdown file of help content for each page in this module.
```

Comment:
It might be good to specify that quirk about the help texts... I think it's that if the help file exists in the DB then the one from the help editor won't be shown? Or it could be the other way around.

Reply:
ok